dev/tests/functional/phpunit.xml.dist

```diff
@@ -6,7 +6,7 @@
  */
 -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

Comment:
[`dev/tests/functional/composer.json`](https://github.com/magento/magento2/blob/c2810e0d6014fd8aa787d24e5d46c6593437f00a/dev/tests/functional/composer.json) suggests that for running functional tests, the dependencies declared therein will be used. Not sure if this is actually the case, but this is the reason I'm using a different location here.